CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Current Trunk
 
  - BinaryenSelect no longer takes a type parameter.
  - AutoDrop APIs have been removed.
+ - Binaryen now supports parsing control flow structures with parameter types.
```

Comment:
```suggestion
 - Binaryen now supports parsing control flow structures with parameter types
   (this is not fully optimized, however, and is therefore not recommended).
```

Reply:
I would expect it to make basically no difference after optimizations, so I'm not sure we need to be this conservative in the changelog note. If producers can get simpler unoptimized modules by using control flow inputs and it basically makes no change to their optimized output, then it's a still a net win.

Reply:
True, yeah, so my wording wasn't good. But still, I can imagine someone does a bunch of work to use inputs to control flow, and expect Binaryen to preserve that benefit, which seems surprising when it doesn't. Maybe a note that we do not emit such shapes?

Reply:
I expanded the note to say that we lower the input parameters away, which I think is the important information here.

Reply:
sgtm though I don't see that pushed?

Reply:
Oops, my bad. Will fix.